src/base/User.ts

```diff
@@ -22,6 +22,7 @@ import UserRole from './UserRole';
 
 interface User extends PersistedModel, Person {
 	role : UserRole;
+	password? : String;
```

Comment:
Not yet. It might become a requirement if project lead decides to ship this product to other planets! I wouldn't be surprised 😝